core/src/com/unciv/models/ruleset/unique/Unique.kt

```diff
@@ -225,7 +225,7 @@ class UniqueMapTyped: EnumMap<UniqueType, ArrayList<Unique>>(UniqueType::class.j
 
 
 // Will probably be allowed to be used as a conditional when I get the motivation to work on that -xlenstra
```

Comment:
This comment should also be removed now that it's implemented :upside_down_face: